pkg/controller/operands/kubevirt.go

```diff
@@ -458,6 +458,9 @@ func getKVDevConfig(hc *hcov1beta1.HyperConverged) (*kubevirtcorev1.DeveloperCon
 	if useKVMEmulation {
 		devConf.UseEmulation = useKVMEmulation
 	}
+	if lv := hc.Spec.LogVerbosityConfig; lv != nil && lv.Kubevirt != nil {
```

Comment:
why also `&& lv.Kubevirt != nil` ?

Reply:
This is because this function is responsible for kubvirt logs only. In this PR the only attribute for `hc.Spec.LogVerbosityConfig` is `Kubevirt`, but In the future log verbosity might be added to more components.

Reply:
ok